nilearn/__init__.py

```diff
@@ -8,67 +8,18 @@
 
 See http://nilearn.github.com for complete documentation.
 """
+import gzip
 
-from version import __version__
+from .version import check_module_dependencies
 
+#
+check_module_dependencies()
+del check_module_dependencies
```

Comment:
I think that rather than importing and deleting, which we did to avoid polluting the namespace, eg for tab completion, I would rename the function to _check_module_dependencies, and not delete it. IPython will by default hide these from tab-completion.